datafusion/Cargo.toml

```diff
@@ -42,12 +42,13 @@ simd = ["arrow/simd"]
 crypto_expressions = ["md-5", "sha2"]
 regex_expressions = ["regex", "lazy_static"]
 unicode_expressions = ["unicode-segmentation"]
+pyarrow = ["pyo3", "libc", "arrow/pyarrow"]
 
 [dependencies]
 ahash = "0.7"
-hashbrown = "0.11"
-arrow = { version = "5.1", features = ["prettyprint"] }
-parquet = { version = "5.1", features = ["arrow"] }
+hashbrown = { version = "0.11", features = ["raw"] }
+arrow = { path = "../../arrow-rs/arrow", features = ["prettyprint"] }
+parquet = { path = "../../arrow-rs/parquet", features = ["arrow"] }
```

Comment:
Path-only dependencies are only fine in a private monorepo. Otherwise, you need both a version and a path.

Reply:
Only used for local prototyping. It also depends on an arrow-rs PR so need to wait that first.